### ISSUE TITLE
Fail to lauch--ImportError: cannot import name 'soft_unicode' from 'markupsafe'

### ISSUE THREAD

**Issue by chunleili** [adamzap/landslide#228]
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (D:\App\miniconda3\lib\site-packages\markupsafe\__init__.py)

The full error message:
```
Traceback (most recent call last):
  File "D:\App\miniconda3\Scripts\landslide-script.py", line 33, in <module>
    sys.exit(load_entry_point('landslide==1.1.8', 'console_scripts', 'landslide')())
  File "D:\App\miniconda3\Scripts\landslide-script.py", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "D:\App\miniconda3\lib\importlib\metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "D:\App\miniconda3\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "D:\App\miniconda3\lib\site-packages\landslide\main.py", line 8, in <module>
    from . import generator
  File "D:\App\miniconda3\lib\site-packages\landslide\generator.py", line 7, in <module>
    import jinja2
  File "D:\App\miniconda3\lib\site-packages\jinja2\__init__.py", line 33, in <module>
    from jinja2.environment import Environment, Template
  File "D:\App\miniconda3\lib\site-packages\jinja2\environment.py", line 15, in <module>
    from jinja2 import nodes
  File "D:\App\miniconda3\lib\site-packages\jinja2\nodes.py", line 19, in <module>
    from jinja2.utils import Markup
  File "D:\App\miniconda3\lib\site-packages\jinja2\utils.py", line 647, in <module>
    from markupsafe import Markup, escape, soft_unicode
```

**Comment by yihuang** [comment]
```
pip3 install MarkupSafe==2.0.1
```

workaround

**Comment by adamzap** [comment]
I just pinned `MarkupSafe` to `v1.1.1` and released `v1.1.9` of `landslide`. Could someone try it and verify that it is now working on a fresh install? It seems to be working on my end.

**Comment by adamzap** [comment]
Please reopen this issue if you are still having a problem.
